src/bilder/images/dagster/templates/vector/dagster_logs.yaml

```diff
@@ -0,0 +1,14 @@
+---
+transforms:
+  parse_dagster_logs:
+    type: remap
+    inputs:
+    - "parse_docker_stdout_stderr"
+    source: |
+      .application = "dagster"
```

Comment:
This is probably going to require another pass to be able to pull in logs from the different pipeline containers. We'll likely want to apply a common label to all of them so that we can set the proper source filtering.

Reply:
So actually under service it will be the name of the docker compose service as written in docker-compose.yaml for example:

```
{
  "application": "dagster",
  "service": "lakehouse-assets-graph",
  "environment": "data-production"
  ...
```

and 
```
{
  "application": "dagster",
  "service": "edx-gcs-courses-pipeline",
  "environment": "data-production"
  ...
```
https://github.com/mitodl/ol-infrastructure/blob/main/src/bilder/images/dagster/templates/docker-compose.yaml.j2

Reply:
https://github.com/mitodl/ol-infrastructure/pull/1328/files#diff-4869c0991783135c961f7f43ce25ca110148cc7c7fa2c621528c17036e3f43efR16